crypto/mem.c

```diff
@@ -172,7 +173,8 @@ void *CRYPTO_malloc(size_t num, const char *file, int line)
 {
     INCREMENT(malloc_count);
     if (malloc_impl != CRYPTO_malloc)
-        return malloc_impl(num, file, line);
+        return ERR_raise_malloc_failure(malloc_impl(num, file, line),
+                                        file, line, NULL);
```

Comment:
You're duplicating the raise call here, aren't you?

Reply:
What exactly do you mean here? I cannot spot any duplication here.

Reply:
Because the raise call is done in the OPENSSL_malloc macro already. CRYPTO_malloc is not supposed to be called directly.